pkg/epinio/detail/applications.vue

```diff
@@ -215,7 +215,7 @@ export default Vue.extend<Data, any, any, any>({
 
     commitPosition() {
       if (!this.preparedCommits.length && !this.gitDeployment.deployedCommit) {
-        return;
+        return null;
```

Comment:
i'm not sure these changes do anything, commitPosition is only used in a single place. the prop it's passed to converts the undefined to defaults, and within the component it's just `v-if="commitPosition"` anyway?

Reply:
Yes, we can remove this fix sorry.